src/main/java/no/nav/syfo/service/NaermesteLedersMoterService.java

```diff
@@ -15,21 +21,28 @@
 
     private MoteService moteService;
 
+    private PdlConsumer pdlConsumer;
+
     @Inject
     public NaermesteLedersMoterService(
             NarmesteLederConsumer narmesteLederConsumer,
-            MoteService moteService
+            MoteService moteService,
+            PdlConsumer pdlConsumer
     ) {
         this.narmesteLederConsumer = narmesteLederConsumer;
         this.moteService = moteService;
+        this.pdlConsumer = pdlConsumer;
     }
 
-    public List<Mote> hentNaermesteLedersMoter(String nlAktoerId) {
-        List<NarmesteLederRelasjon> narmesteLederRelasjoner = narmesteLederConsumer.narmestelederRelasjonerAnsatte(nlAktoerId);
+    public List<Mote> hentNaermesteLedersMoter(String lederAktorId) {
```

Comment:
Blanding av ae og a som erstatning for æ, vi har vel landa på a? Eller er det oppe til diskusjon igjen 😁

Reply:
Nei, det er vel ikke noen ny diskusjon der, bare jeg som ikke tenkte på å endre navnet da jeg holdt på 👍